tests/features/asset_distribution/add_solution.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: "Add asset distribution" visibility options.
+  In order to manage distributions
+  As a moderator
+  I need to be able to add "Asset distribution" rdf entities through UI.
+
+  Scenario: "Add distribution" button should only be shown to moderators.
+    Given the following collection:
+      | name | Asset Distribution Test                  |
+      | uri  | https://a.distribution/solution/add/test |
+      | logo | logo.png                                 |
```

Comment:
As a user, when creating a collection I cannot determine the URI, so this should not be presented as an option.